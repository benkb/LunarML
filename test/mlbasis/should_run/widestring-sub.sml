print (WideChar.toString (WideString.sub ("foo", 0)) ^ "\n");
print (WideChar.toString (WideString.sub ("foo", 1)) ^ "\n");
print (WideChar.toString (WideString.sub ("foo", 2)) ^ "\n");
print (WideChar.toString (WideString.sub ("foo", ~1)) ^ "\n" handle Subscript => "Subscript\n");
print (WideChar.toString (WideString.sub ("foo", 3)) ^ "\n" handle Subscript => "Subscript\n");
print (WideChar.toString (WideString.sub ("foo", 4)) ^ "\n" handle Subscript => "Subscript\n");
