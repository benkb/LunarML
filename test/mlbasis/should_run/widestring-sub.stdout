f
o
o
Subscript
Subscript
Subscript
